server/src/config/index.js

```diff
@@ -0,0 +1,18 @@
+const { Pool } = require('pg')
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+console.log(process.env.DATABASE_URL);
+
+const pool = new Pool({ 
+  connectionString : process.env.DATABASE_URL,
+})
+
+module.exports = {
```

Comment:
why are you making use of es5 export instead of es6?

Reply:
Changes made